sealed-secrets/README.md

```diff
@@ -0,0 +1,9 @@
+sealed-secrets
+==============
+
+This directory provides a Dockerfile to build a Docker container for [sealed-secrets](https://github.com/bitnami-labs/sealed-secrets#kustomize).
```

Comment:
The fragment `#kustomize` in the URL seems to have no importance.

Reply:
Thanks, updated.